ansible_collections/arista/cvp/docs/_build/cv_change_control_v3.rst

```diff
@@ -69,7 +69,18 @@ The following options may be specified for this module:
     <td></td>
     <td></td>
     <td>
-        <div>The name of the change control. If not provided, one will be generated.</div>
+        <div>The name of the change control, If not provided, one will be generated automatically.</div>
```

Comment:
@JulioPDX how's this sentenceless format exactly? For new thoughts, do we capitalize the first letter or not? Like in this case should it be `If` or `if`?